net-misc/ipsvd/ipsvd-1.0.0-r3.ebuild

```diff
@@ -0,0 +1,51 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```

Comment:
Sorry but this won't do anymore, this is too old. You should update to EAPI 8 while at it this time.